app/web/src/features/communities/events/constants.ts

```diff
@@ -1,53 +1,8 @@
 import { EventsType } from "queryKeys";
 
-export const ATTENDEES = "Attendees";
 export const details = ({ colon = false }: { colon?: boolean } = {}) =>
   `Details${colon ? ":" : ""}`;
```

Comment:
Looking at the usage of this, I suspect we'll probably want to duplicate this so that we have `Details` as its own standalone constant, and have `Details: {eventDetails}` as its own interpolation, since some languages might have two different words for this depending on the context, and so we probably want that flexibility as opposed to coupling ourselves against the English implementation by over DRYing this. What do you think?

(Feel free to leave this if you are not up for removing these functions as part of this)

Reply:
Yeah, I'm definitely postponing this change until I finish the easy ones. But I agree that this can be solved with interpolation.

Reply:
Agree, I'll address this in another PR